portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -17,30 +17,65 @@
 import com.google.gson.Gson;
 import java.util.ArrayList;
 import java.io.IOException;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static final String textParameter = "text-input";
+  private static final String nameParameter = "name";
+  private static final String timestampParameter = "timestamp";
+
```

Comment:
Constants should be in UPPER_SNAKE_CASE.

Reply:
+1 Whoops, sorry for the incorrect instructions earlier.